glam/api/models.py

```diff
@@ -267,3 +267,19 @@ class Meta:
                 fields=["channel", "build_id"],
             )
         ]
+
+
+class UsageInstrumentation(models.Model):
+    ACTION_PROBE_SEARCH = 'PROBE_SEARCH'
+    ACTIONS = [
+        (ACTION_PROBE_SEARCH, "probe_search"),
+    ]
+    id = models.AutoField(primary_key=True)
```

Comment:
If you think you'll eventually get a lot of data in this table, you might consider `models.BigAutoField` instead.